primitives/avn-common/src/event_discovery.rs

```diff
@@ -54,33 +68,43 @@ impl Ord for DiscoveredEvent {
     }
 }
 
+pub type FractionsCount = u16;
 type EthEventsPartition = BoundedBTreeSet<DiscoveredEvent, EventsBatchLimit>;
 #[derive(PartialEq, Eq, Clone, Encode, Decode, Debug, TypeInfo, MaxEncodedLen)]
 pub struct DiscoveredEthEventsFraction {
-    data: EthEventsPartition,
-    fraction: u16,
-    fraction_count: u16,
     id: H256,
+    fraction: FractionsCount,
```

Comment:
IMO this makes it hard to read the code when compared to `fraction: u16`

Reply:
I agree with Nahu. The meaning of 'fractions count' should be communicated by the variable name, not by the type. The type is more like 'units' of measurement, if you like, and I think it would only be relevant if we had lots of operations between several variables of the same 'type'

Reply:
Main reason I added this is this:
https://github.com/AventusProtocolFoundation/avn-parachain/pull/371/files#diff-b16d6c46e1465c34530cee63d8a1b6cafdf9cd5e55e481ef95c83202734895bbR132
It improves readability on the data struct.
I can remove though if that is the preference